cdap-app-fabric-tests/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowStatsSLAHttpHandlerTest.java

```diff
@@ -63,13 +65,30 @@
   private static MetricStore metricStore;
   private static Store store;
 
+  private int sourceId;
+
   @BeforeClass
   public static void beforeClass() throws Throwable {
     AppFabricTestBase.beforeClass();
     store = getInjector().getInstance(DefaultStore.class);
     metricStore = getInjector().getInstance(MetricStore.class);
   }
 
+  private void setStartAndRunning(final ProgramId id, final String pid, final long startTime) {
```

Comment:
instead of copying these to every test case that needs them, it may be better to 
- either add  them to AppFabricTestBase (if all tests extend from that)
- or create a class with two static methods that can be called from all tests